tests/tests.targets

```diff
@@ -48,6 +48,9 @@
       <XunitArgs>$(XunitArgs) @(IncludeTraitsItems->'-trait %(Identity)', ' ')</XunitArgs>
       <XunitArgs>$(XunitArgs) @(ExcludeTraitsItems->'-notrait %(Identity)', ' ')</XunitArgs>
 
+      <!-- Color output doesn't work well when capturing the output in the CI system -->
+      <XunitArgs>$(XunitArgs) -nocolor</XunitArgs>
```

Comment:
This is unfortunate, this is quite useful for local testing.

Reply:
It would be nice to have an option passed through from runtest.py if possible.

Reply:
I can add an issue to put it back under a switch. I figure the annoyance of having it in logs is much greater than not having it when running locally -- especially since I wonder how many people run locally.

Reply:
I think it is worth having it under a switch; however, I do not think it is worth blocking this pr for it.

Reply:
Resolving as this should not block the pr